extension/wrapper.cpp

```diff
@@ -95,9 +95,12 @@ static PyObject *
 Satrec_twoline2rv(PyTypeObject *cls, PyObject *args)
 {
     char *string1, *string2, line1[130], line2[130];
-    double dummy;
+    gravconsttype whichconst = wgs72;
+    double dummy;    
```

Comment:
This is a bit confusing — why does it think you are adding and removing the line `double dummy;` even though it doesn't look different? It turns out that trailing whitespace has accidentally worked its way into the line. I have just added a check for trailing whitespace to CI so that this gets caught automatically in the future to save contributors time:

6a4ca1d854cb3337f6a7b5eb3c5cf48a3e73f5ed

If you'll rebase atop the latest master, it should help you get the file back into good shape!